tests/core/process.c

```diff
@@ -340,8 +342,9 @@ PROCESS_TESTCASE (process_ranges_exclude_cloaked)
   g_assert (!ctx.found);
 }
 
-#ifdef HAVE_DARWIN
+#if defined (G_OS_WIN32) || defined (HAVE_DARWIN)
 
+#define TEST_STACK_BUFFER_SIZE 50
```

Comment:
Let's add a blank line after this define. Defines shouldn't be stuck to functions following them.